pychunkedgraph/ingest/ran_agglomeration.py

```diff
@@ -185,6 +223,7 @@ def _collect_edge_data(imanager: IngestionManager, chunk_coord):
     edge_dtype = cg_meta.edge_dtype
     subfolder = "chunked_rg"
     path = f"{imanager.config.AGGLOMERATION}/{subfolder}/"
+    path = imanager.config.AGGLOMERATION
```

Comment:
I do not think this should be a permanent change

Reply:
yes I will revert that after I build image for this particular run